docs/topics/api/auth.rst

```diff
@@ -1,7 +1,109 @@
+.. _api-auth:
+
 ================
 Authentication
 ================
 
-To authenticate against this API you will need an API key.
+To access the API, you need to include a `JSON Web Token (JWT)`_
+in an authorization header for every request. This header authenticates
+your user account so you could think of it like a session cookie.
+
+Access Credentials
+==================
+
+To create JWTs, first obtain a **key** and **secret** from the
+`API Credentials Management Page`_.
+Keep the secret string well protected and never check it into version control.
+If someone obtains your secret they can make API requests on behalf of your
+user account.
+
+Create a JWT for each request
+=============================
+
+Prior to making every API request, you need to generate a fresh `JWT`_.
+The JWT will have a short expiration time and is only valid for a single
+request so you can't cache or reuse it.
+You only need to include a few standard fields; here's what the raw JSON object
+needs to look like before it's signed:
+
+.. code-block:: json
+
+    {
+        "iss": "user:543210:23",
+        "iat": 1447273096,
+        "exp": 1447273156,
+    }
+
+iss
+    This is a `standard JWT claim`_ identifying
+    the *issuer*. Set this to the **API key** you generated on the
+    `credentials management page`_.
+    For example: ``user:543210:23``.
+iat
+    This is a `standard JWT claim`_ indicating
+    the *issued at time*. It should be a Unix epoch timestamp and
+    **must be in UTC time**.
+exp
+    This is a `standard JWT claim`_ indicating
+    the *expiration time*. It should be a Unix epoch timestamp in UTC time
+    and must be **no longer than 30 seconds** past the issued at time.
+
+.. note::
+    If you're having trouble authenticating, make sure your system
+    clock is correct and consider synchronizing it with something like
+    `NTP (Network Time Protocol) <http://www.ntp.org/>`_.
```

Comment:
Acronyms go in the parentheses again but in this case I'd be tempted to just say `NTP` since it is probably only ever referenced once and that's likely what people know.